sidebars.js

```diff
@@ -165,6 +165,12 @@ module.exports = {
             'react-native-testing-library/setup',
           ],
         },
+        {
```

Comment:
Could you also add a redirect to this page at https://github.com/testing-library/testing-library-docs/blob/main/netlify.toml#L85 please?

Reply:
done :)